src/UserManager.ts

```diff
@@ -263,7 +263,11 @@ export class UserManager {
     }
 
     protected async _useRefreshToken(state: RefreshState): Promise<User> {
-        const response = await this._client.useRefreshToken(state);
+        const response = await this._client.useRefreshToken({
+            state,
+            timeoutInSeconds: this.settings.silentRequestTimeoutInSeconds
+                ? Math.max(this.settings.silentRequestTimeoutInSeconds - 2, 1) : 8,
```

Comment:
Why is this set to subtract 2 seconds from `silentRequestTimeoutInSeconds`?

Reply:
You are right makes no sense, in this scenario it is actually `silentRequestTimeoutInSeconds`. I will fix that when i do the rebase.